public/style/main.css

```diff
@@ -161,88 +170,110 @@ img {
     text-align: center;
 }
 
-.warnings>h5{
-    font-size:1.25vw;
+.warnings>h5 {
+    font-size: 1.25vw;
 }
 
-@media(max-width:700px){
-    .box-1{
+@media(max-width:700px) {
+    .box-1 {
         width: 70vw;
     }
-    .info{
-        width:45vw;
+
+    .info {
+        width: 45vw;
     }
-    img{
+
+    img {
         width: 80%;
     }
-    .warnings>h5{
+
+    .warnings>h5 {
         font-size: 8.8125px;
-        
+
     }
+
     .exp-text>p {
         font-size: 10.374px !important;
         text-align: center;
     }
 
 }
 
-@media(max-width:800px){
-    .info{
+@media(max-width:800px) {
+    .info {
         flex-direction: column;
         align-items: center;
         justify-content: space-between;
     }
 }
 
-.buttons-div{
+.buttons-div {
     margin-bottom: 1.5%;
 }
 
-.btn-back{
-    padding:5px 1vw 5px 1vw;
-    
+.btn-back {
+    padding: 5px 1vw 5px 1vw;
+
 }
 
-.btn-next{
-    padding:5px 2.7vw 5px 2.7vw !important; 
-    background-color:#1277db;
-    color:white;
+.btn-next {
+    padding: 5px 2.7vw 5px 2.7vw !important;
+    background-color: #1277db;
+    color: white;
 
 }
 
-.badge-img{
+.badge-img {
     display: flex;
     flex-direction: row;
     justify-content: space-around;
-    margin-top:2%;
+    margin-top: 2%;
 }
 
 .input-img-op {
     margin-top: 5%;
 
     background-color: #ebe8e8;
-    border:solid 2px #d8d8d8;
+    border: solid 2px #d8d8d8;
     display: flex;
     justify-content: center;
     align-items: center;
     flex-direction: column;
 }
 
-.result{
+.result {
     display: flex;
     flex-direction: column;
     justify-content: center;
     margin: 4% 0 4% 0;
 }
 
-.result>h2,.result>h4{
+.result>h2,
+.result>h4 {
     font-family: 'Nanum Gothic', sans-serif;
     text-align: center;
-    margin:3% 0 3% 0;
-    font-weight:bold;
+    margin: 3% 0 3% 0;
+    font-weight: bold;
     color: rgb(105, 105, 105);
 }
 
 #drop-area.highlight {
     border-color: purple;
-  }
\ No newline at end of file
+}
+
+.faded-out {
+    opacity: 0;
+}
+
+#loading-1,#loading-2{
+    background: url('../images/spinner.gif') no-repeat center center;
+     position: absolute;
+    top: 0;
+    left: 0; 
+    height: 100%;
+    width: 100%;
+    z-index: 9999999;
```

Comment:
That `z-index` tho. Is there any "prettier" way of doing this?

Reply:
I am not sure. Actually adopted this.